inline-c-objc/inline-c-objc.cabal

```diff
@@ -38,3 +38,6 @@ test-suite tests
   default-language:    Haskell2010
   if os(darwin)
     frameworks:        Foundation
+    buildable:         True
+  else
+    buildable:         False
```

Comment:
Linux does not have the Foundation library, so it can not be tested.

Reply:
In theory Linux could use GNUstep, I suppose, but let's keep it simple.